### PR TITLE
New page to proxy network html page

### PR DESCRIPTION
**Pull request by alisman**
We need to load network HTML page inside of iframe.  This proxy allows us to work around cross origin iframe restrictions 

# Checks
- [ ] Runs on Heroku.
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to master.

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

### CODE REVIEW

File: portal/src/main/webapp/networkProxy.jsp [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<c:import url="https://rawgit.com/alisman/cbioportal-frontend/gitNet/src/pages/resultsView/network/network.htm" />
```
Comment (alisman) [inline portal/src/main/webapp/networkProxy.jsp]: i will change this to file hosted on main repo

File: portal/src/main/webapp/networkProxy.jsp [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<c:import url="https://rawgit.com/alisman/cbioportal-frontend/netFrame/src/pages/resultsView/network/network.htm" />
```
Comment (inodb) [inline portal/src/main/webapp/networkProxy.jsp]: maybe set this to specific commit after merging https://github.com/cBioPortal/cbioportal-frontend/pull/1290/files ?

Comment (alisman) [reply portal/src/main/webapp/networkProxy.jsp]: will do

**Comment by alisman** [comment]
@inodb this became just an update of frontend version so that network.htm would be there

**Pull request merged**